src/resources/espnow.cc

```diff
@@ -32,19 +32,88 @@
 #include "../event_sources/system_esp32.h"
 #include "../event_sources/ev_queue_esp32.h"
 
-#define ESPNOW_RX_DATAGRAM_NUM      8
 #define ESPNOW_RX_DATAGRAM_LEN_MAX  250
-#define ESPNOW_TX_WAIT_US           1000
+#define ESPNOW_RX_DATAGRAM_NUM      8
```

Comment:
I'd probably drop the alignment of the values now that they are so far apart due to the comment.